### ISSUE TITLE
Remove "n people use this"

### ISSUE THREAD

**Issue by theluk** [jimrhoskins/ngmodules.org#24]
I think from a marketing perspective this one is not the best.
If your website would have somethink like github, where you click it and it gets you a local copy or something to your drive, i would use "n people use this". 
For me it seems like almost nobody (or very few people) uses this website. 
The question is not always who uses it. But who would like to use it. As we know, concepts like "star", "favorite", "like" are better, as it is easier to say I like it than I use it.

**Comment by adyba** [comment]
Well, I don't really about marketing aspects of the wording but I'm one of the few (?!?) using this site regularly. As there is 600+ modules I would trade "n people use this" for a  "my personal selection" anytime. The chance to see a Misko's module selection would be the best scoring system ;)

**Comment by mwawrusch** [comment]
What about scanning signed up users github repos for bower.json and use the modules referenced there. Gives much more accurate data and removes the need for the  I use this button altogether.

**Comment by rpocklin** [comment]
Yeah i've had many PRs for my angular module and yet `1 person` uses it.  It's not accurate at all.
If you pulled in # stars and # forks from github that would be more accurate.
